omero/training-server/playbook.yml

```diff
@@ -124,6 +150,16 @@
         group: "omero-server"
         force: yes
 
+    - name: Download the batch_roi_export_to_table.py script
+      become: yes
+      get_url:
+        url: https://raw.githubusercontent.com/will-moore/training-scripts/b398d7526e48df79a6c33ce61a679d1458a15ded/practical/python/server/batch_roi_export_to_table.py
```

Comment:
I assume this URL will be updated to point to the organizational repository? Also does this effectively define a dependency between prod-playbooks and training-scripts down the line i.e. will more training-server scripts be included?

Reply:
It should be https://raw.githubusercontent.com/ome/training-scripts/master/practical/python/server/batch_roi_export_to_table.py
now that the PR has been merged